CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Fix type error when `result_file_type` is expected to be a `FileType` but is `None`
 - Fix chunking for elements with None text that has AttributeError 'NoneType' object has no attribute 'strip'.
 - Invalid elements IDs are not visible in VLM output. Parent-child hierarchy is now retrieved based on unstructured element ID, instead of id injected into HTML code of element.
+- Fix bs4 deprecation warnings by updating `findAll()` with `find_all()`.
```

Comment:
Could you also bump the change log version above and the one in `__version__.py` please? Thank you!